libs/ledger/include/ledger/chaincode/wallet_record.hpp

```diff
@@ -172,37 +172,43 @@ struct WalletRecord
       it = cooldown_stake.erase(it);
     }
   }
+};
+
+}  // namespace ledger
+
+namespace serializers {
+template <typename D>
+struct ArraySerializer<ledger::WalletRecord, D>
```

Comment:
Can we raise an issue for switching this to a map record

Reply:
Yes